Dockerfile

```diff
@@ -4,15 +4,13 @@ MAINTAINER Andy Driver <andy.driver@digital.justice.gov.uk>
 
 WORKDIR /home/cpanel
 
-ADD app app/
-ADD bin bin/
 ADD package.json package.json
-
 RUN npm install
-ADD node_modules node_modules/
+
+ADD app app/
+ADD bin bin/
 
 RUN npm run-script collect-static
-ADD static static/
```

Comment:
I think we might need `static/` because possibly not all static files are generated.

Reply:
Aren't they? I tried to delete `static` and run `npm run-script collect-static` and as far as I could tell `git status` didn't see any change (that's why I assumed everything in `static` was generated off other files)

A question for our own JS/node ninja @axemonkey probably :)